libs/features/mas/src/mas-element.js

```diff
@@ -166,7 +168,8 @@ export class MasElement {
         if (options) this.options = options;
```

Comment:
I wonder if you can listen the very first `mas:pending` event.
If you try to register an even listener for a `<span is="price" data-wcs-osi="xyz">/span>`, it is possible that, the `pending` event will be dispatched before you register. 
To check. I would really remove pending all together, and consider the default state as pending.

Reply:
removed it, agree it's an unnessary event

Reply:
i would somewhere here though add some debug log to mention that element is pending resolution

Reply:
added, thx